src/test/eo/org/eolang/collections/list-tests.eo

```diff
@@ -439,3 +439,35 @@
   assert-that > @
     list3.at 0
     $.equal-to 1
+
+[] > nested-reduced
```

Comment:
@EugeneDar may be its better to name it `nested-reduced-1` because the next is
`nested-reduced-2`

Reply:
@levBagryansky I think it will be not very beautiful and I didn't seen such namings before.